### DISCUSSION TITLE
How to remove grey border framing graph?

### DISCUSSION THREAD

**Discussion by nigeljw3** [alandefreitas/matplotplusplus#289] [General]
Hi,

All the images in the docs have a white background outside the graph, but I get an ugly grey color fill in that area. How can I switch this to white?

<img width="280" alt="test" src="https://user-images.githubusercontent.com/504090/184004985-80337a0a-f933-4845-bc8d-a0523bf54c38.PNG">

Cheers,
Nigel

**Comment by alandefreitas** [comment]
IIRC, this is the default background. It's grey for the interactive terminal and usually transparent for other backgrounds.

There should be some property to set that: https://github.com/alandefreitas/matplotplusplus/blob/ccb4715304c001de9a0aa94a67cc43899ec7a36d/source/matplot/core/figure_type.h#L208-L256
